runtime/criusupport/criusupport.cpp

```diff
@@ -982,7 +982,10 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		if (NULL != vm->checkpointState.restoreArgsList) {
 			/* mark -Xoptionsfile= as consumed */
 			FIND_AND_CONSUME_ARG(vm->checkpointState.restoreArgsList, STARTSWITH_MATCH, VMOPT_XOPTIONSFILE_EQUALS, NULL);
-			bool dontIgnoreUnsupportedRestoreOptions = FIND_AND_CONSUME_ARG(vm->checkpointState.restoreArgsList, EXACT_MATCH, VMOPT_XXIGNOREUNRECOGNIZEDRESTOREOPTIONSENABLE, NULL) < 0;
+			UDATA ignoreEnabled = FIND_AND_CONSUME_ARG(vm->checkpointState.restoreArgsList, EXACT_MATCH, VMOPT_XXIGNOREUNRECOGNIZEDRESTOREOPTIONSENABLE, NULL);
```

Comment:
```suggestion
			IDATA ignoreEnabled = FIND_AND_CONSUME_ARG(vm->checkpointState.restoreArgsList, EXACT_MATCH, VMOPT_XXIGNOREUNRECOGNIZEDRESTOREOPTIONSENABLE, NULL);
```
https://github.com/eclipse-openj9/openj9/blob/eec6dde8f79fa04eaf246b0d213dd7ec3334768f/runtime/oti/j9nonbuilder.h#L4751

Similar suggestion for `UDATA ignoreDisabled` below.